scopes/ui-foundation/use-box/dropdown/dropdown.module.scss

```diff
@@ -1,5 +1,9 @@
 @import '~@teambit/base-ui.layout.breakpoints/_breakpoints.scss';
```

Comment:
can be removed from this PR 
Will be fixed in #6714 

https://github.com/teambit/bit/pull/6714/files#diff-c6ad0b4f888ffd34e5951ae088072825f01723ad2e8388179ccc9b756e948b88